### DISCUSSION TITLE
Codepsaces randomly drops ports?

### DISCUSSION THREAD

**Discussion by jassmith** [community/community#5840] [Codespaces]
My ports are forwarded via my .devcontainer configuration. Sometimes for no obvious reason Codespaces will just clear every forwarded port I have, which can be very confusing when the service worker takes over and serves me stale files. I then have to go back to the ports tab and re-add all my ports.

Anybody else getting this?

**Comment by bryhasagithub** [comment]
Yeah bro, same here!

I feel hella silly because I'll be working on stale files and not realize it for too long

**Comment by derekbekoe** [comment]
We've pushed an update to the [GitHub Codespaces VS Code extension](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces) (v1.0.7) that fixes an issue with ports disappearing.

Let us know if you continue to see issues here.

**Reply by jassmith** [reply]
We still see issues regularly. Now more often what we see is the ports are reported as working but are not actually working.
